Sources/Apodini/WebService.swift

```diff
@@ -57,10 +62,17 @@ extension WebService {
     }
     
 
-    /// This function is provided to start up an Apodini `WebService`. The `app` parameter can be injected for testing purposes only. Use `WebService.main()` to startup an Apodini `WebService`.
-    /// - Parameter app: The app instance that should be injected in the Apodini `WebService`
-    static func main(app: Application) {
-        let webService = Self()
+    /**
+     This function is provided to start up an Apodini `WebService`. The `app` parameter can be injected for testing purposes only. Use `WebService.start()` to startup an Apodini `WebService`.
+     - Parameters:
+         - app: The app instance that should be injected in the Apodini `WebService`
+         - webService: The instanciated `WebService` by the Swift ArgumentParser
+     */
+    static func start(app: Application, webService webServiceTemp: Self? = nil) {
+        /// If `WebService` isn't already instanciated by the Swift ArgumentParser, manually create an instance here
+        let webService = webServiceTemp ?? Self()
+        
```

Comment:
I think that we can simplify the signature a bit:
```suggestion
    static func start(app: Application, webService: Self = Self()) {
```